src/manager.cpp

```diff
@@ -347,8 +347,100 @@ types::DbusVariantType
 void Manager::collectSingleFruVpd(
     const sdbusplus::message::object_path& i_dbusObjPath)
 {
-    // Dummy code to supress unused variable warning. To be removed.
-    logging::logMessage(std::string(i_dbusObjPath));
+    try
```

Comment:
I did not see the API restricting collection based on the flag which says whether the FRU is collectable or not.

Reply:
I raised the concern about CM flag during flowchart creation if I remember. If it is not there then I guess we missed that. 
The name of flag can be something other than CM but a flag should be there to denote which all FRUs qualify for getting collected via this API.

For the time being,please create a single defect and log that here in the comment to
1) Add flag in JSON.
2) Add that in the flow diagram
3) Update the code here.

Reply:
https://github.com/ibm-openbmc/openpower-vpd-parser/issues/349
https://github.com/ibm-openbmc/openpower-vpd-parser/issues/350

two issues has been created to update the flag in JSON and in Manager's collectSingleFRUVPD api. 
flow chart updated in box folder .